services/src/test-library/src/main/java/org/openkilda/testing/service/kafka/KafkaBreakerImpl.java

```diff
@@ -45,17 +63,9 @@ private void setState(KafkaBreakTarget target, KafkaBreakerAction action) throws
         ProducerRecord<String, String> record = new ProducerRecord<>(
                 topic, target.toString(), action.toString());
         try {
-            for (int attempt = 0; attempt < MAX_ATTEMPTS; attempt += 1) try {
-                logger.debug(
-                        "Send {} to {} ({} of {}})",
-                        record.value(), target.toString(), attempt + 1, MAX_ATTEMPTS);
-                producer.send(record).get();
-                break;
-            } catch (InterruptedException e) {
-                logger.warn("producer was interrupted (attempts {} of {})", attempt, MAX_ATTEMPTS);
-            }
-        } catch (ExecutionException e) {
-            throw new KafkaBreakException("Unable to publish control message", e);
+            producer.send(record).get();
```

Comment:
In the previous variant, we had a log message. It looks like now we don't do any logging. Or I am missing something?

Reply:
Ok, thanks.